deepchecks/nlp/checks/data_integrity/text_property_outliers.py

```diff
@@ -81,10 +81,17 @@ def run_logic(self, context: Context, dataset_kind: DatasetKind) -> CheckResult:
         for name, values in properties.items():
             # If the property is single value per sample, then wrap the values in list in order to work on fixed
             # structure
+            is_numeric = name not in cat_properties
+            property_error = None
             if not isinstance(values[0], list):
-                values = [[x] for x in values]
+                if is_numeric:
+                    curr_nan_count = pd.isnull(values).sum()
+                    values = pd.to_numeric(values, errors='coerce')
+                    updated_nan_count = pd.isnull(values).sum()
+                    if updated_nan_count > curr_nan_count:
+                        property_error = 'Numeric property contains non-numeric values.'
```

Comment:
would here raise exception - no need to run outlier calculation for properties like this

Reply:
Done